extensions/amp-story/amp-story.md

```diff
@@ -446,6 +446,36 @@ src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/a
   </noscript>
 </amp-img>
 
+**AMP-to-AMP linking**
+
+You can use AMP-to-AMP linking in the components that support a url by including `'amphtml': true` inside the component linking to another AMP document. For example:
```

Comment:
This doesn't explain what AMP-to-AMP linking. How about:

"For documents displayed in an AMP viewer, links typically navigate `_top` or open in a new window. Links to AMP pages, however, may continue to be displayed in the viewer. To enable this behavior, add `"amphtml": true` to the link object. For example:"

/cc @bpaduch